assembl/fabfile.py

```diff
@@ -794,6 +801,155 @@ def update_pip_requirements(force_reinstall=False):
         run("yes w | %s" % cmd)
 
 
+def is_db_updated():
+    """
+    Return if the database is update or not
+    """
+    history = venvcmd('alembic -c %s history' % (env.ini_file))
+    current = venvcmd('alembic -c %s heads' % (env.ini_file))
+    return current in history
+
+
+@task
+def reset_db():
+    """
+    Restore and update the latest database
+    """
+    # Only for the staging server (for tests)
+    if env.wsginame == 'staging.wsgi':
+        exists = False
+        # Test if the dump exists on the amazon s3 object storage
+        with shell_env(
+            AWS_ACCESS_KEY_ID=env.aws_access_key_id,
+            AWS_SECRET_ACCESS_KEY=env.aws_secret_access_key
+        ):
+            exists = venvcmd('aws s3 ls s3://%s/%s | wc -l' % (env.aws_bucket_name, get_db_dump_name()))
+            exists = False if exists == '0' else True
+
+        print(green('Restore and update the latest database'))
```

Comment:
Can we use a real logger. https://docs.python.org/2/library/logging.html
This backup should be run automatically and we would like to be able to debug the logs if they don't run properly.

Reply:
je suis pour un éclatement et un nettoyage globale du fabfile... à discuter